envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/RBACFilterFactory.kt

```diff
@@ -114,25 +114,36 @@ class RBACFilterFactory(
             loggedEndpointsPolicies.values
         )
 
-        val shadowPolicies = commonPolicies + loggedEndpointsPolicies
-        val actualPolicies = commonPolicies + allowUnlistedPolicies
+        val shadowPolicies = (commonPolicies + loggedEndpointsPolicies)
```

Comment:
I may be missing something but isn't the other version just a workaround for invalid classification of `allowed_client`? This version looks actually simpler to me but as I said, I may be missing something.